crmprtd/moti/download.py

```diff
@@ -35,7 +37,7 @@
 
 def verify_date(datestring, default, label="start_time"):
```

Comment:
Since we're now using an external library to verify that user inputted dates are parsable and interpretable into a datetime object, I'd recommend that we *remove* the `verify_date()` function.

That said, `dateutil.parser`'s errors are not exactly user-friendly to the users of this application. E.g.:

```python
>>> from dateutil.parser import parse
>>> parse("Jan 10, 2000")
datetime.datetime(2000, 1, 10, 0, 0)
>>> parse("11/11/11")
datetime.datetime(2011, 11, 11, 0, 0)
>>> parse("not a date")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/james/.local/share/virtualenvs/crmprtd-Sg6rF-Cq/lib/python3.8/site-packages/dateutil/parser/_parser.py", line 1358, in parse
    return DEFAULTPARSER.parse(timestr, **kwargs)
  File "/home/james/.local/share/virtualenvs/crmprtd-Sg6rF-Cq/lib/python3.8/site-packages/dateutil/parser/_parser.py", line 649, in parse
    raise ValueError("Unknown string format:", timestr)
ValueError: ('Unknown string format:', 'not a date')
```
So maybe instead of removing it altogether, we could move it out of `moti.download`, to `crmprtd.download` and then use it (consistently) across *all* of the download modules? Then each module that needs time parsing could use our internal function which either parses it and returns the parsed result, or gives a user-friendly warning and sets the default.